### PR TITLE
added npm dev task

### PR DESCRIPTION
**Pull request by svengabr**
You can now run your local server using `npm run dev` instead of `npm start`.

Source maps will be created making debugging a lot easier.
Javascript won't be minimized.

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@svengabr 

Does the command `npm run dev` also build the web before it runs the server? Maybe you could also make that clear in the README changes.

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ npm start
 
 - `npm install` installs the necessary packages to build, display and test the website on your local system. You only need to run this once or if the packages defined in [package.json](package.json) are changed.
 
-- The command `npm start` triggers the commands `npm run build` followed by `npm run start-server` described below and which can be used individually if desired.
+- The command `npm start` triggers the commands `npm run build` followed by `npm run start-server` described below and which can be used individually if desired. However, assets like Javascript and CSS will be compressed and minified using that method. If you want to debug you can start the local server using the command `npm run dev`.
```
Comment (MikeMcC399) [inline README.md:44]: Suggest to make it a bit clearer:

"If you want to debug you can build the web and start the local server using the command `npm run dev` as an alternative to `npm start`."

Comment (svengabr) [reply README.md]: @MikeMcC399 Thanks, I have changed the readme text accordingly

**Comment by svengabr** [comment]
@MikeMcC399 yes its the same as `npm start` but with `--dev` flag in order to trigger the PRODUCTION constant to false 

> ```const PRODUCTION = !(yargs.argv.develop || yargs.argv.dev);```

This will disable compression of scripts and have webpack build the sourcemaps.

This is primarily useful when you want to attach the debugger from the dev tools.

debug

**Review by larswmh — APPROVED** [review]

**Pull request merged**